examples/given_when_then.rs

```diff
@@ -1,22 +0,0 @@
-
-extern crate rspec;
```

Comment:
keep the example

Reply:
The [README](https://github.com/rust-rspec/rspec/pull/30/files#diff-04c6e90faac2675aa89e2176d2eec7d8) now contains a paragraph (`### Suites, Contexts & Examples`) on the different styles. Sufficient? No? Comments?

Reply:
Yes the readme is great, but there is never enough small, simple, focused examples. :)